dxr/plugins/clang/tests/test_user_defined_literals.py

```diff
@@ -0,0 +1,23 @@
+from dxr.plugins.clang.tests import CSingleFileTestCase
+
+
+class UserDefinedLiteralTests(CSingleFileTestCase):
+    cflags = '-std=c++11'
+
+    source = """
+        int operator "" _i(const char *) { return 0; }
+
+        int main() {
+          42_i;
+          return 0;
+        }
+        """
+
+    def test_def(self):
+        self.found_line_eq('+function:"operator""_i(const char *)"',
```

Comment:
Yes, this is fine. It turns out I even wrote out my rationale: https://github.com/mozilla/dxr/blob/c7f124a0cf1c68c7b7a8941c33b66a4883154f9e/dxr/query.py#L255-L263